front/app/containers/Admin/formBuilder/components/FormBuilderTopBar/index.tsx

```diff
@@ -77,8 +77,8 @@ const FormBuilderTopBar = ({ isSubmitting }: FormBuilderTopBarProps) => {
       top="0px"
     >
       <Box
-        p="16px"
-        w="212px"
+        p="15px"
```

Comment:
@benfraserdesign Normally we try to stick with paddings/margins as multiples of 4 (to align with any future "spacing" component in the component library). Is 15px and 210px definitely needed here?